### ISSUE TITLE
[Features of Eval Tools Section] Add sentence back in

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-selecting-eval-tools#3]
[Strong]
In the opening paragraphs after the section title "Features of Eval Tools", I think the statement, "which may help users to compare and assess Web accessibility evaluation tools for their specific needs." should be added back into this area. 

I think that is an important statement explaining why the features are important and called out in the database. They help with making the decision of which tools may or may not work for an individual or project.

**Comment by shawna-slh** [comment]
+1

**Comment by lakeen** [comment]
Done.
